assets/js/phoenix_live_view.js

```diff
@@ -526,11 +526,19 @@ export class LiveSocket {
       this.owner(e.target, view => view.submitForm(e.target, phxEvent))
     }, false)
 
+    // https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/change_event
+    // https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/input_event
+    // Chrome and Firefox diverge here. On first input, "change" is fired in Chrome
+    // but not in Firefox
     for(let type of ["change", "input"]){
       this.on(type, e => {
         let input = e.target
-        let key = input.type === "checkbox" ? "checked" : "value"
+        let inputType = input.type
+        let key = inputType === "checkbox" ? "checked" : "value"
         if(this.prevInput === input && this.prevValue === input[key]){ return }
+        // When mid input with a `,` or `.`, Chrome reports the value as "" as it doesn't pass their validation.
+        // As a result, we should not do anything mid state
+        if(inputType === "number" && (e.data === "." || e.data === ",")){ return }
```

Comment:
There are tradeoffs here.  We don't get any server sent events.

This is happening at the earliest place possible.  We could do this much later and just avoid patching the dom.  This would have to happen in the `channel.push` callback.

Reply:
Also I should mention `step="0.01"` doesn't fix the issue either.  I would have thought it did